django/urls/resolvers.py

```diff
@@ -235,7 +235,7 @@ def _route_to_regex(route, is_endpoint=False):
         except KeyError as e:
             raise ImproperlyConfigured(
                 "URL route '%s' uses invalid converter %s." % (original_route, e)
-            )
+            ) from e
```

Comment:
Here too -- I'd prefer to see `raw_converter` replace `e` in the exception message

Reply:
Agreed and fixed.

Reply:
And basically that's how it works even now, because `msg` for `KeyError` contains a nonexistent key. We can use `raw_converter`.